src/rendering/loadingscreen.cpp

```diff
@@ -134,6 +136,13 @@ LoadingScreen::LoadingScreen(ShowMessage showMessage, ShowNodeNames showNodeName
         );
     }
 
+    _logFont = global::fontManager->font(
+        "Loading",
+        LogFontSize,
```

Comment:
```suggestion
        LogFontSize * global::windowDelegate->osDpiScaling(),
```

Also need to `#include <openspace/engine/windowdelegate.h>` at the top